src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -60,20 +61,20 @@ protected override async Task OnInitializedAsync()
 
         var loadingTcs = new TaskCompletionSource();
 
-        await TrackResourceSnapshotsAsync();
+        TrackResourceSnapshots();
 
         // Wait for resource to be selected. If selected resource isn't available after a few seconds then stop waiting.
         try
         {
-            await loadingTcs.Task.WaitAsync(TimeSpan.FromSeconds(3));
+            await loadingTcs.Task.WaitAsync(TimeSpan.FromSeconds(5));
```

Comment:
This has always bothered me, but as a user I would have expected that the selection defaults to the first option when not otherwise specified from the route. When the page loads, it would ideally show logs (obviously, I could select something else) - am I alone in that expectation?